src/Symfony/Bundle/FrameworkBundle/Resources/config/translation.php

```diff
@@ -158,5 +160,17 @@
             ->args([service(ContainerInterface::class)])
             ->tag('container.service_subscriber', ['id' => 'translator'])
             ->tag('kernel.cache_warmer')
+
+        ->set('translation.provider_collection', TranslationProviderCollection::class)
+            ->factory([service('translation.provider_collection_factory'), 'fromConfig'])
+            ->args([
+                [], // TranslationProviderCollection
```

Comment:
not sure that the comment provides anything (I don't understand it personally...)

Reply:
shouldn't this be an abstract arg then ?